src/coreComponents/linearAlgebra/utilities/LinearSolverParameters.hpp

```diff
@@ -124,6 +124,9 @@ struct LinearSolverParameters
 #endif
     integer useAdaptiveTol = false;   ///< Use Eisenstat-Walker adaptive tolerance
     real64 weakestTol = 1e-3;         ///< Weakest allowed tolerance when using adaptive method
+    real64 strongestTol = 1e-8;       ///< Strongest allowed tolerance when using adaptive method
+    real64 adaptiveGamma = 0.1;       ///< Gamma parameter for adaptive method
+    real64 adaptiveExponent = 1.0;    ///< Exponent parameter for adaptive method
```

Comment:
The default values --  `exponent = 2.0` and `gamma = 0.9` -- values have changed. I understand that the old exponent is probably too high. How did you choose the new values?

Reply:
Yes, you are right.
I just found that exponent=2 is too aggressive but at the same time 0.9 is too weak.
Found 1.0 and 0.1 are more balanced but could be case dependent, so I pulled them out into the input.